k8s/elastic-gke-logging/README.md

```diff
@@ -172,6 +172,16 @@ If you use a different namespace than the `default`, run the command below to cr
 kubectl create namespace "$NAMESPACE"
 ```
 
+#### Create Fluentd service account
+
+Create Fluentd service account and role binding
+
+```shell
+export FLUENTD_SERVICE_ACCOUNT="$APP_INSTANCE_NAME-fluentdserviceaccount"
+kubectl create serviceaccount --namespace $NAMESPACE $FLUENTD_SERVICE_ACCOUNT
+kubectl create clusterrolebinding $FLUENTD_SERVICE_ACCOUNT-rule --clusterrole=view --serviceaccount=$NAMESPACE:$FLUENTD_SERVICE_ACCOUNT
```

Comment:
The permissions seems to be higher than required. Could we create a SA with the permission described in `schena.yaml`?

https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/elastic-gke-logging/schema.yaml#L37-L50

You can use [RabbitMQ](https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/rabbitmq#expand-the-manifest-template) as an example.

Reply:
Are you sure about this example? It is binding cluster-admin role, which gives much more permissions